models/ticket_history/int_zendesk__field_history_scd.sql

```diff
@@ -18,10 +18,15 @@ with change_data as (
         ticket_day_id
 
         {% for col in ticket_columns if col.name|lower not in ['date_day','ticket_id','ticket_day_id'] %} 
+            {% if target.type == 'postgres' %}
+                ,last_value( {{ col.name }} ) over
+                    (partition by ticket_id order by case when date_day is not null then 0 else 1 end asc, date_day rows between unbounded preceding and current row) as {{ col.name }} 
 
-        ,last_value({{ col.name }} ignore nulls) over 
-          (partition by ticket_id order by date_day asc rows between unbounded preceding and current row) as {{ col.name }}
+            {% else %}
+                ,last_value({{ col.name }} ignore nulls) over 
+                    (partition by ticket_id order by date_day asc rows between unbounded preceding and current row) as {{ col.name }}
 
+            {% endif %}
```

Comment:
This is one of the two only conditionals that were required for Postgres compatibility of the package. For the Postgres conditional I was split between two different statements to achieve `ignore nulls` as Postgres does not support this within window functions. 

The other statement I was between was the below:
```
,last_value( {{ col.name }} ) over
                    (partition by ticket_id order by case when date_day is not null then 0 else 1 end asc, date_day rows between unbounded preceding and current row) as {{ col.name }} 
```

I felt the coalesce was less confusing and probably more accurate. Let me know if you would like to discuss.